packages/web3-core/types/index.d.ts

```diff
@@ -201,8 +201,30 @@ export interface Log {
 }
 
 export interface Content {
-    pending: any;
-    queued: any;
+    pending: TxPoolContent;
+    queued: TxPoolContent;
+}
+
+export interface TxPoolContent {
+    [address: string]: {
+        [nonce: number]: Array<Transaction>
```

Comment:
@nivida Just now I and @joshstevens19 had a discussion that for consistency we should use Transaction[ ] not Array<Transaction>.

Reply:
Funny I had it first defined with ``Transaction[]`` and saw above that we were using ``Array<type>``. I'll go through all types and will change it to ``type[]``. Thanks! 💪